### DISCUSSION TITLE
Scrolling in github mobile, black flashes

### DISCUSSION THREAD

**Discussion by lucidliving69** [community/community#134313] [Mobile]
I noticed an issue while scrolling in github mobile (android apk, version 1.171.0)

When scrolling past gifs, the screen flashes black rapidly. 

https://github.com/user-attachments/assets/7b68361c-0679-428b-a815-163747e5b3ff

**Comment by elisealix22** [comment]
Hi @lucidliving69, thank you so much for providing the video! I opened up an issue to investigate this internally. We will look into it!

**Comment by Loveco255** [comment]
> I noticed an issue while scrolling in github mobile (android apk, version 1.171.0)
> 
> When scrolling past gifs, the screen flashes black rapidly. 
> 
> https://github.com/user-attachments/assets/7b68361c-0679-428b-a815-163747e5b3ff
> 
>

**Comment by kangarruu** [comment]
@lucidliving69 we've been unable to reproduce this issue, has this resolved for you or are you still seeing it?

**Comment by lucidliving69** [comment]
I'm still seeing it.

**Comment by kangarruu** [comment]
Thanks for the update! Would you mind sharing what phone and Android version you're using?

**Reply by lucidliving69** [reply]
Samsung Galaxy S22 Ultra - Model: SM-S908W, Stock Android 14, OneUI 6.1

Github app version 1.175.0

Edit: Looks like it only happens with the MagSpoof repo. It may be a gif issue.

**Reply by kangarruu** [reply]
Thanks for those details!
